src/emulator/controller.ts

```diff
@@ -350,6 +353,17 @@ export async function startAll(options: any, noGui: boolean = false): Promise<vo
     await startEmulator(hostingEmulator);
   }
 
+  if (shouldStart(options, Emulators.LOGGING)) {
```

Comment:
Since it doesn't actually matter what port this is on, I think the logging emulator should start at `<some base port>` and scan up until it finds a free port.  As implemented I think it will fail if this exact port is not free.